src/main/java/seedu/mindmymoney/command/UpdateCommand.java

```diff
@@ -60,7 +60,9 @@ public void executeCommand() throws MindMyMoneyException {
             }
             String amountString = expenditureDescription.substring(divisionIndex).strip();
             int indexToUpdate = Integer.parseInt(indexString) + LIST_INDEX_CORRECTION;
-            Expenditure newExpenditure = new Expenditure(description, category, Integer.parseInt(amountString));
+            //to edit to fit new add command
```

Comment:
I see this comment in a few other places in the code. If the implementation of the new add command is complete, you may remove it.

Reply:
its meant for the others to change! when they edit their code they have reminder